src/connector/with_options_sink.yaml

```diff
@@ -402,9 +402,14 @@ MqttConfig:
     comments: Whether the message should be retained by the broker
     required: false
     default: Default::default
-  - name: r#type
+  - name: topic.field
     field_type: String
-    required: true
+    required: false
+  - name: fallback_to_default
```

Comment:
Due to the same reason in my prev comment, no need to introduce `fallback_to_default`.